docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -358,21 +358,21 @@ Configures a generic BPMN element property.
 
 #### `zeebe:input`
 
-| **Binding `type`**          | `zeebe:input`                                         |
-| --------------------------- | ----------------------------------------------------- |
-| **Valid property `type`'s** | `String`<br /> `Text`<br />`Hidden`<br />`Dropdown`   |
-| **Binding parameters**      | `name`: The name of the input parameter               |
-| **Mapping result**          | `<zeebe:input target="[name]" source="[userInput] />` |
+| **Binding `type`**          | `zeebe:input`                                                                                                                                              |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Valid property `type`'s** | `String`<br /> `Text`<br />`Hidden`<br />`Dropdown`<br />`Boolean`<br />`Number`<br />(`Boolean` and `Number` require `feel` to be `optional` or `static`) |
```

Comment:
We don't need the part in parentheses anymore. Let's mark the default in the `feel` section.

Reply:
Cf. https://github.com/camunda/camunda-docs/pull/4904/commits/9827fea9cfc6c871aec1ae47902cb2273996215f

Reply:
Yes, I'll update.